lib/routes/shared/security_pin/backup_phrase/verify_backup_phrase_page.dart

```diff
@@ -158,7 +167,7 @@ class VerifyBackupPhrasePageState extends State<VerifyBackupPhrasePage> {
             elevation: 0.0,
             shape: const StadiumBorder(),
             onPressed: () {
-              if (_formKey.currentState.validate()) {
+              if (_formKey.currentState.validate() && !_hasError) {
```

Comment:
Shouldn't we reset` _hasError = false` before calling `_formKey.currentState.validate()`

Reply:
[We do](https://github.com/breez/breezmobile/commit/a3e77fbe8f9ddbdba67d41599da0179db000d0ef), this file seems outdated.